### PR TITLE
DEV-4190 Enabled xml documentation

### PR DESCRIPTION
**Pull request by steco**

### CODE REVIEW

File: sdk/Lusid.Sdk/Utilities/ApiConfiguration.cs [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+        /// </summary>
         public string Username { get; set; }
+
+        /// <summary>
+        /// Password
+        /// </summary>
         public string Password { get; set; }
+
+        /// <summary>
+        /// Client identifier
```
Comment (rizwansaeed) [inline sdk/Lusid.Sdk/Utilities/ApiConfiguration.cs]: OAuth2 Client ID

File: sdk/Lusid.Sdk/Utilities/ApiConfiguration.cs [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
+        /// </summary>
         public string ClientId { get; set; }
+
+        /// <summary>
+        /// Client secret
```
Comment (rizwansaeed) [inline sdk/Lusid.Sdk/Utilities/ApiConfiguration.cs]: OAuth2 Client Secret

File: sdk/Lusid.Sdk/Utilities/ApiExceptionExtensions.cs [hunk 3]
```diff
@@ -5,6 +5,9 @@
 
 namespace Lusid.Sdk.Utilities
 {
+    /// <summary>
+    /// Utility functions
```
Comment (rizwansaeed) [inline sdk/Lusid.Sdk/Utilities/ApiExceptionExtensions.cs]: Extension to deserialise LUSID exceptions contained within the ApiException.ErrorContent

File: sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs [hunk 4]
```diff
@@ -4,8 +4,14 @@
 
 namespace Lusid.Sdk.Utilities
 {
+    /// <summary>
+    /// Builder class to build instances of ILusidApiFactory
+    /// </summary>
     public class LusidApiFactoryBuilder
     {
+        /// <summary>
+        /// Create an ILusidApiFactory using the specified secrets.json file
```
Comment (rizwansaeed) [inline sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs]: ~secrets.json~ - it can be any file name

Comment (steco) [reply sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs]: Should we create a kb article on secrets.json format ?

File: sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs [hunk 5]
```diff
@@ -10,7 +10,7 @@ namespace Lusid.Sdk.Utilities
     public class LusidApiFactoryBuilder
     {
         /// <summary>
-        /// Create an ILusidApiFactory using the specified secrets.json file
+        /// Create an ILusidApiFactory using the specified configuration file.  For
```
Comment (rizwansaeed) [inline sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs]: Rogue 'For'

Comment (steco) [reply sdk/Lusid.Sdk/Utilities/LusidApiFactoryBuilder.cs]: Added a link to the kb

**Pull request merged**